src/app/loadout/LoadoutPopup.tsx

```diff
@@ -308,14 +308,12 @@ function LoadoutPopup({
                 </span>
               )}
             </li>
-            {!hideFarming && (
```

Comment:
I don't follow this one - what is hideFarming even for?

Reply:
It was added with Active Mode 
https://github.com/DestinyItemManager/DIM/pull/5969 @kyleshay

Reply:
it's bool checked upstream already is all i know